src/components/posts/PostLike.js

```diff
@@ -0,0 +1,67 @@
+import like from './like.png';
+import React from 'react';
+import { useState, useEffect } from 'react';
+import client from '../../utils/client';
+import './style.css';
+import colours from '../../utils/colours';
+
+const PostLike = ({likes=[], postId}) => { 
+
+const blue = colours.blue;
```

Comment:
The other way to do this could be to set the class name, and have the colors defined in the CSS class name? Rather than  mix of some styles in code and some styles in CSS. Not worth changing just now if you're wanting to get this PR across the line but possible refactoring in future.